File: src/main/java/com/jesperancinha/service/pojos/Runway.java
package com.jesperancinha.service.pojos;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * Created by joaofilipesabinoesperancinha on 01-08-16.
 */
@Getter
@Setter
@NoArgsConstructor
public class Runway {

    private String id;

    private String airportRef;

    private String airportIdent;

    private String lengthFt;

    private String widthFt;

    private String surface;

    private String lighted;

    private String closed;

    private String leIdent;

    private String leLatitudeDeg;

    private String leLongitudeDeg;

    private String leElevationFt;

    private String leHeadingDegT;

    private String leDisplacedThresholdFt;

    private String heIdent;

    private String heLatitudeDeg;

    private String heLongitudeDeg;

    private String heElevationFt;

    private String heHeadingDegT;

    private String heDisplacedThresholdFt;

    private String heDisplacedThresholdFT;

}

File: airports-app/src/main/kotlin/org/jesperancinha/airports/pojos/Runway.kt
package org.jesperancinha.airports.pojos

/**
 * Created by joaofilipesabinoesperancinha on 01-08-16.
 */
data class Runway(
    val id: String? = null,
    val airportRef: String? = null,
    val airportIdent: String? = null,
    val lengthFt: String? = null,
    val widthFt: String? = null,
    val surface: String? = null,
    val lighted: String? = null,
    val closed: String? = null,
    val leIdent: String? = null,
    val leLatitudeDeg: String? = null,
    val leLongitudeDeg: String? = null,
    val leElevationFt: String? = null,
    val leHeadingDegT: String? = null,
    val leDisplacedThresholdFt: String? = null,
    val heIdent: String? = null,
    val heLatitudeDeg: String? = null,
    val heLongitudeDeg: String? = null,
    val heElevationFt: String? = null,
    val heHeadingDegT: String? = null,
    val heDisplacedThresholdFt: String? = null,
    val heDisplacedThresholdFT: String? = null
)